go.mod

```diff
@@ -108,8 +108,9 @@ require (
 	golang.org/x/net v0.25.0
 	golang.org/x/sync v0.7.0
 	golang.org/x/sys v0.20.0
+	golang.org/x/text v0.16.0
 	golang.org/x/time v0.5.0
-	golang.org/x/tools v0.21.0
+	golang.org/x/tools v0.21.1-0.20240508182429-e35e4ccd0d2d
```

Comment:
any reason to use un-tagged version here? I would have expected no version change is required for this change (except moving x/text from indirect to direct dependency block).

Reply:
hey @sayboras this got added while updating the cases, how should i resolve, and also i raised another pull request removing the cases dependency, as it is a change of 15k files, as suggested by joe i have just added nolint directive as strings.Title is included in go V1 API promises, can you please refer to #32936 instead, i will close this PR if another PR i mentioned serves the purpose or i can continue using cases dependency